shared/js/devtools/rollouts.js

```diff
@@ -0,0 +1,82 @@
+import { getAssignedCohortSettingsKey, getRolloutSettingsKey } from '../background/components/remote-config';
+
+async function updateSettingAndReloadConfig(settingKey, settingValue) {
+    await chrome.runtime.sendMessage({
+        messageType: 'updateSetting',
+        options: { name: settingKey, value: settingValue },
+    });
+    await chrome.runtime.sendMessage({ messageType: 'forceReprocessConfig' });
+}
+
+const table = document.querySelector('table');
+/** @type {HTMLTemplateElement | null} */
+const rowTemplate = document.querySelector('template#rollout-row');
+
+async function render() {
+    if (!table || !rowTemplate) {
+        throw new Error('Missing HTML elements');
+    }
+
+    for (const row of table.querySelectorAll('.subfeature')) {
+        table.removeChild(row);
+    }
+    /** @type {import("../background/components/remote-config").SubFeatureStatus[]} */
+    const rolloutStatus = await chrome.runtime.sendMessage({ messageType: 'getSubfeatureStatuses' });
+    rolloutStatus.forEach((f) => {
+        const row = rowTemplate.content.cloneNode(true);
+        const cells = row.querySelectorAll('td');
+        cells[0].innerText = `${f.feature} / ${f.subFeature}`;
+        cells[1].classList.add(f.state);
+        cells[1].innerText = f.state;
+        cells[2].innerText = f.hasTargets ? 'Yes' : 'No';
+        cells[3].innerText = f.rolloutPercent ? f.rolloutPercent : '-';
+        cells[4].innerText = f.rolloutRoll ? Math.floor(f.rolloutRoll) : '-';
+        cells[5].innerText = f.hasCohorts && f.cohort ? f.cohort.name : '-';
```

Comment:
Mixing ternary and boolean operators like this is easy to screw up in my experience. Would you mind adding parenthesis around the condition part (same below), just to make it super clear? E.g. 
```suggestion
        cells[5].innerText = (f.hasCohorts && f.cohort) ? f.cohort.name : '-';
```

Reply:
Prettier is removing these brackets after I add them, so I think this change would lead to a lint error.